db/seeds.rb

```diff
@@ -6,3 +6,4 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+User.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
```

Comment:
It seems to me this line doesn't work. Did you execute it in rails console?
Probably, it raise: first_name can't be blank, and last_name can't be blank

Reply:
I've changed it and tested it, it worked!